can/io/blf.py

```diff
@@ -25,24 +25,32 @@
 from can.util import len2dlc, dlc2len
 
 
+class BLFParseError(Exception):
```

Comment:
Should this extend `IOError`?

Reply:
I googled it and the [accepted answer](https://stackoverflow.com/questions/9276663/what-exception-class-to-use-for-file-parsing-error) seems to be to inherit from Exception. IOError is more low level I think.

Reply:
Okay, I'm convinced. ;)